src/component/mxgraph/shape/gateway-shapes.ts

```diff
@@ -73,6 +54,23 @@ abstract class GatewayShape extends mxRhombus {
     const yTranslation = parentY + parentHeight / 4;
     c.translate(xTranslation, yTranslation);
   }
+
+  protected paintCrossIcon(canvas: MxScaleFactorCanvas, parentWidth: number, parentHeight: number): void {
+    canvas.begin();
+    canvas.moveTo(parentWidth * 0.38, 0);
```

Comment:
:warning: usage of scaling in this method shows that the canvas wrapper is not fully used
I suggest to manage this in another refactoring PR